### PR TITLE
Dp 6841 header splitsing

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/header/services/header-size/header-size.constant.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const headerSize = {
```
Comment (royderks) [inline src/header/services/header-size/header-size.constant.js]: Use uppercase for constants

File: src/app/components/Header/Header.test.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Header from './Header';
+
+describe('Header', () => {
```
Comment (royderks) [inline src/app/components/Header/Header.test.jsx]: Add test cases for all three versions of the header

File: src/app/App.jsx [hunk 3]
```diff
@@ -188,7 +189,9 @@ App.propTypes = {
   printModeLandscape: PropTypes.bool.isRequired,
   embedPreviewMode: PropTypes.bool.isRequired,
   overflowScroll: PropTypes.bool.isRequired,
-  user: PropTypes.object.isRequired // eslint-disable-line react/forbid-prop-types
+  user: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```
Comment (royderks) [inline src/app/App.jsx]: Write as PropTypes.shapeOf({})

File: src/header/services/header-size/header-size.constant.js (hunk 1 above)
Comment (aditudorache) [reply src/header/services/header-size/header-size.constant.js]: done!

File: src/app/components/Header/Header.test.jsx (hunk 2 above)
Comment (aditudorache) [reply src/app/components/Header/Header.test.jsx]: sure! thanks!

File: src/app/App.jsx (hunk 3 above)
Comment (aditudorache) [reply src/app/App.jsx]: fixed in the whole project!

**Pull request merged**